### PR TITLE
Makefile.streamlit

### PR DESCRIPTION
**Pull request by jdu**
Adds a new Makefile.streamlit for building and deploying the streamlit container to ECR for deployment to the cluster.

### CODE REVIEW

**Comment by jdu** [comment]
This can be used by passing the path to the Makefile, I made a separate Makefile as it's only concerned with building the docker container for streamlit and not with anything else really. 

You can run the commands in it by passing the `-f` arg to `make`

i.e.

```
make -f Makefile.streamlit docker-push
```

File: Makefile.streamlit [hunk 1]
```diff
@@ -0,0 +1,20 @@
+AWS_ACCOUNT_ID := 160358319781
```
Comment (nsorros) [inline Makefile.streamlit:1]: note that the repo is public

**Review by nsorros — APPROVED** [review]
LGTM 👍 Note the some env variables you are exposing. In fact the user is instructed to have some of them in their env https://github.com/wellcometrust/grants_tagger/blob/master/README.md#-env-variables

**Comment by jdu** [comment]
Ah right, I forgot this repo is public, it might make sense then to move this Makefile elsewhere as it's specific to our infrastructure.

**Comment by nsorros** [comment]
What should we do with this @jdu?

File: Makefile.streamlit (hunk 1 above)
Comment (jdu) [inline Makefile.streamlit:1]:
```suggestion
AWS_ACCOUNT_ID := ${AWS_ACCOUNT_ID}
```

File: Makefile.streamlit [hunk 2]
```diff
@@ -0,0 +1,20 @@
+AWS_ACCOUNT_ID := ${AWS_ACCOUNT_ID}
+ECR_IMAGE := 160358319781.dkr.ecr.eu-west-1.amazonaws.com/org.wellcome/labs
```
Comment (jdu) [inline Makefile.streamlit:2]:
```suggestion
ECR_IMAGE := ${AWS_ACCOUNT_ID}.dkr.ecr.eu-west-1.amazonaws.com/org.wellcome/labs
```

**Pull request merged**